### PR TITLE
feature(wedlocks, email-templates): Adding more template bases for emails

### PR DESCRIPTION
**Pull request by julien51**
# Description

We use a different template base for emails sent for events.

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/email-templates/src/index.ts [hunk 1]
```diff
@@ -63,4 +63,4 @@ Object.keys(EmailTemplates).forEach((template: string) => {
 const getEmailTemplate = (template: Template | string) => templates[template]
 
 export default templates
-export { getEmailTemplate, base }
+export { getEmailTemplate, bases }
```
Comment (searchableguy) [inline packages/email-templates/src/index.ts:66]: I'm a little worried about this change since base is not under bases which may break something.

**Review by searchableguy — APPROVED** [review]

File: wedlocks/src/route.js [hunk 2]
```diff
@@ -6,15 +6,14 @@ import templates from '@unlock-protocol/email-templates'
...
-console.log({ config })
-
 /**
  * Builds the template and params
  * @param {*} args
  * @returns
  */
 const getTemplateAndParams = async (args, opts) => {
   let template = templates[args.template.toLowerCase()]
+  console.log(template)
```
Comment (julien51) [inline wedlocks/src/route.js:16]:
```suggestion
```

File: packages/email-templates/src/index.ts (hunk 1 above)
Comment (julien51) [reply packages/email-templates/src/index.ts:66]: Not sure what you mean?

Comment (searchableguy) [reply packages/email-templates/src/index.ts:66]: the exports are changed. If you want to import base now, you have to import it from under bases.

```ts
import { bases } from "./templates.ts"
const { base } = bases
```

**Pull request merged**